submit.py

```diff
@@ -172,7 +188,8 @@ def write_glidein_part(self, f, local_dir=None, glidein_tarball=None,
         if "CustomEnv" in self.config:
             for k, v in self.config["CustomEnv"].items():
                 f.write(k + '=' + v + ' ')
-        f.write('./%s\n' % glidein_script)
+        self.write_line(f, 'cd $LOCAL_DIR')
```

Comment:
There's already a `cd` several lines up.  And you can't put that inside the `env -i` command.